docs/modules/ROOT/pages/security/security-fix-list.adoc

```diff
@@ -9,6 +9,8 @@ can or have impacted Payara Server across releases:
 |=======================================================================
 |ID |CVSS v3.0 Base Score |Status |Summary |Release |Observations
 
+|https://nvd.nist.gov/vuln/detail/CVE-2021-42392[CVE-2021-42392] | N/A | N/A | Unauthenticated RCE in H2 Database Console |  | Unrelated to Payara Server. H2 Database Console doesn't launch the H2 Database Console and doesn't make it available in any way.
```

Comment:
Sentence doesn't seems to be correct

**H2 Database Console** doesn't launch the **H2 Database Console** ...

Reply:
You're right, I'll fix it.
```suggestion
|https://nvd.nist.gov/vuln/detail/CVE-2021-42392[CVE-2021-42392] | N/A | N/A | Unauthenticated RCE in H2 Database Console |  | Unrelated to Payara Platform. The Payara Platform doesn't launch the H2 Database Console and doesn't make it available in any way.
```